x/perpetual/keeper/begin_blocker.go

```diff
@@ -66,7 +66,13 @@ func (k Keeper) BeginBlocker(ctx sdk.Context) {
 				FundingRateLong:    fundingRateLong,
 				FundingRateShort:   fundingRateShort,
 			})
+
+			err = k.UpdateNetOpenInterest(ctx, &pool)
```

Comment:
@fenriz07 I would refrain from adding any unnecessary computation at the ABCI level (begin/end blocker), can you instead do this computation within the query function, another option would be to calculate that value within an offchain indexer. cc @Wgil @amityadav0 @avkr003

Reply:
agree, we can add this in query, also we don;t need to store it as well. So no migration is needed